sphinxlint/__main__.py

```diff
@@ -109,6 +109,12 @@ def __call__(self, parser, namespace, values, option_string=None):
         help="comma-separated list of fields used to sort errors by. Available "
         f"fields are: {SortField.as_supported_options()}",
     )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=" ".join(["sphinxlint", __version__]),
+        help="print version",
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "-V", "--version", action="version", version=f"Sphinx Lint {__version__}"
    )
```
Or we could write the name as it appears in the README:
```console
❯ sphinx-lint --version
Sphinx Lint 0.6.7
```
This is similar to at least Homebrew:
```console
❯ brew --version
Homebrew 4.1.2
```